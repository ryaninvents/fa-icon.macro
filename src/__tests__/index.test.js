import pluginTester from 'babel-plugin-tester';
import plugin from 'babel-plugin-macros';

pluginTester({
  plugin,
  snapshot: true,
  babelOptions: { filename: __filename },
  tests: [
    `
      import createIcon from './macro';

      const homeIcon = createIcon('./home.svg');
    `,
  ],
});
