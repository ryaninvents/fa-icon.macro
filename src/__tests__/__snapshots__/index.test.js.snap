// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `
"
import createIcon from './macro';

const homeIcon = createIcon('./home.svg');

      ↓ ↓ ↓ ↓ ↓ ↓

const homeIcon = {
  \\"prefix\\": \\"far\\",
  \\"icon\\": [256, 256, [], null, \\"M165.9604 0.0000L165.9604 28.2058L129.3353 3.3868L0.0000 89.2231L37.0090 89.2231L37.0090 256.0000L100.3403 256.0000L100.3403 148.7045L176.2611 148.7045L176.2611 256.0000L223.1889 256.0000L223.1889 89.2231L256.0000 89.2231L210.9791 58.7135L210.9791 0.0000L165.9604 0.0000z\\"]
};
"
`;
